src/app/public/modules/phone-field/phone-field-adapter.service.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  Injectable,
+  Renderer2,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs/Subject';
+
+import {
+  SkyLibResourcesService
+} from '@skyux/i18n';
+
+@Injectable()
+export class SkyPhoneFieldAdapterService implements OnDestroy {
+
+  private ngUnsubscribe = new Subject();
+
+  constructor(
+    private renderer: Renderer2,
+    private resourcesService: SkyLibResourcesService
+  ) { }
+
+  public ngOnDestroy() {
+    this.ngUnsubscribe.next();
+    this.ngUnsubscribe.complete();
+  }
+
+  public addElementClass(element: HTMLElement, className: string) {
```

Comment:
The public methods on this service are missing output types.

Reply:
Fixed